libs/primitives/src/lib.rs

```diff
@@ -249,6 +249,9 @@ pub mod constants {
 	pub const fn deposit(items: u32, bytes: u32) -> Balance {
 		items as Balance * 15 * CENTI_CFG + (bytes as Balance) * 6 * CENTI_CFG
 	}
+
+	/// The prefix for tokens managed by Connectors.
+	pub const GENERAL_CURRENCY_INDEX_PREFIX: [u8; 12] = *b"CfgCnctCurId";
```

Comment:
It might be overkill but I am always supporting safer approaches. Unfortunately we lose out on the `const` property due to using `blake2_128` and splitting. WDYT? [d774926](https://github.com/centrifuge/centrifuge-chain/pull/1292/commits/d774926f35f09d6d1653379f025b26f257199850)